src/MyEventEmitter.js

```diff
@@ -1,14 +1,58 @@
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.events = {};
+  }
+  on(name, callback) {
+    if (!this.events[name]) {
+      this.events[name] = [];
+    }
+    this.events[name].push(callback);
+  }
+  once(name, callback) {
+    const wrapper = (...args) => {
+      callback(...args);
+      this.off(name, wrapper);
+    };
```

Comment:
The `wrapper` function is correctly implemented to ensure the callback is only called once. However, ensure that the `off` method is correctly removing the wrapper function by reference, as this is crucial for the `once` and `prependOnceListener` methods to work correctly.